src/documentation/0051-node-buffers/index.md

```diff
@@ -102,40 +103,40 @@ Just like you can access a buffer with an array syntax, you can also set the con
 
 ```js
 const buf = Buffer.from('Hey!')
-buf[1] = 111 //o
+buf[1] = 111 //o in UTF-8
 console.log(buf.toString()) //Hoy!
 ```
 
-### Copy a buffer
+### Slice a buffer
 
-Copying a buffer is possible using the `copy()` method:
+If you want to create a partial visualization of a buffer, you can create a slice. A slice is not a copy: the original buffer is still the source of truth. If that changes, your slice changes.
+
+Use the `subarray()` method to create it. The first parameter is the starting position, and you can specify an optional second parameter with the end position:
 
 ```js
 const buf = Buffer.from('Hey!')
-let bufcopy = Buffer.alloc(4) //allocate 4 bytes
-buf.copy(bufcopy)
+buf.subarray(0).toString() //Hey!
+const slice = buf.subarray(0, 2)
+console.log(slice.toString()) //He
+buf[1] = 111 //o
+console.log(slice.toString()) //Ho
 ```
 
-By default you copy the whole buffer. 3 more parameters let you define the target buffer starting position to copy to, the source buffer starting position to copy from, and the new buffer length:
+### Copy a buffer
+
+Copying a buffer is possible using the `set()` method:
 
 ```js
 const buf = Buffer.from('Hey!')
-let bufcopy = Buffer.alloc(2) //allocate 2 bytes
-buf.copy(bufcopy, 0, 0, 2)
-bufcopy.toString() //'He'
+let bufcopy = Buffer.alloc(4) //allocate 4 bytes
+bufcopy.set(buf)
 ```
 
-### Slice a buffer
-
-If you want to create a partial visualization of a buffer, you can create a slice. A slice is not a copy: the original buffer is still the source of truth. If that changes, your slice changes.
-
-Use the `slice()` method to create it. The first parameter is the starting position, and you can specify an optional second parameter with the end position:
+By default you copy the whole buffer. If you only want to copy a part of the buffer, you can use `.subarray()` from above and the `offset` argument that specifies an offset to write to:
```

Comment:
```suggestion
By default you copy the whole buffer. If you only want to copy a part of the buffer, you can use `.subarray()` show above and the `offset` argument that specifies an offset to write to:
```

Reply:
I don’t quite understand this one, to be honest – `show above`?

Reply:
Yeah, that was a typo 😆 fixed.